### ISSUE TITLE
Commander Pro sensors stuck at 25 degrees C

### ISSUE THREAD

**Issue by hst12** [iJacks1980/FanControl.CommanderPRO#15]
Using version 115 of Fan Control and 1.0.4 of the Commander Pro plugin the temperature sensors 1 to 3 (4 not plugged in) are all reading 25 C on my system

Checking in iCue, the sensors report normally with a slight variation in each sensor at idle.

TIA for any assistance.

**Comment by iJacks1980** [comment]
You mention iCUE - are you using this at the same time as iCUE is installed?

**Comment by kannan-balasubramanian** [comment]
@hst12 Even if the iCUE services are disabled, if they were launched even once, I suggest restarting the system. I used to face this issue for HWiNFO.

**Comment by hst12** [comment]
I've seen the problem with HWINFO before and avoid using it for that very reason and have also noticed that only a reboot fixes the issue between HWINFO and iCue.

With the system turned off overnight and checking again I see readings of 20, 23 and 23 from the sensors. The 1st sensor is at the front of the case next to the intake fans and usually reads about 2 degrees above ambient temperature.

Fan control on the Commander Pro is working fine as the fans ramp up and down by using core temperature or other sources as input.

I've just run 10 minutes of Cinebench and the sensor values didn't change at all. Its as if the sensors are being read once and not updated again.

**Comment by iJacks1980** [comment]
Hmm, well I can be 100% certain that the code does indeed updated - but that is based on what data it gets back from the Commander PRO.

Currently working on a new version, that is a little broken right now.  However as soon as I fix the (hopefully) last remaining bug I'll post it as a beta release, it saves trace logs so I can see the exact data returned from the sensors.

**Comment by hst12** [comment]
May I suggest adding 1 digit after the decimal point to the temp probe readouts if possible? It would be easier to see if the temps fluctuate at all.

**Comment by iJacks1980** [comment]
That has already been changed in the upcoming 1.0.5 release.

**Comment by hst12** [comment]
Thanks.

**Comment by hst12** [comment]
Thanks.

This is looking good. I can see the differences in sensors.  I’ve attached a log file from this version in case it’s of use.

I see 2 DLLs now. Is that correct? FanControl.Commander.Common and  FanControl.CommanderPro

Regards
Andre

**Comment by iJacks1980** [comment]
Yes, I've done this for my sanity ;)  As there is no way (by default) for a FanControl plugin to not be enabled it meant that every system was trying to load the module for PRO, CORE and CORE XT - they are now separate.

FYI when you reply via email GitHub strips any attachments.

**Comment by hst12** [comment]
Thanks. Let me know if you still need the log file. From what I can see the CoPro is working nicely. The fans are howling while I run MS Flight Simulator but the system is cool. 😄

**Comment by iJacks1980** [comment]
No problem - if it's working fine then it's great.  I've got some more work to do before a full 1.0.5 release, should be just a few days.

Yeah, Flight Sim is the new Crysis ;)

**Comment by artjsalina5** [comment]
Im still getting intermittent 0C NaN as the value for my temperature probe.

**Comment by iJacks1980** [comment]
> Im still getting intermittent 0C NaN as the value for my temperature probe.

You mentioned in another thread that you had a Commander CORE - this thread is about the Commander PRO.

Which device are you actually using?

**Comment by artjsalina5** [comment]
Commander Core. Sorry for the confusion. iCue is not installed.

**Comment by iJacks1980** [comment]
ok, thanks for the clarification.  Can you please raise this as a new issue and include any error/trace logs.
